v2/elasticsearch-common/src/main/java/com/google/cloud/teleport/v2/elasticsearch/transforms/WriteToElasticsearch.java

```diff
@@ -95,6 +96,42 @@ public PDone expand(PCollection<String> jsonStrings) {
             .withConnectionConfiguration(config)
             .withMaxBatchSize(options().getBatchSize())
             .withMaxBatchSizeBytes(options().getBatchSizeBytes());
+    
+    if (options().getJavascriptIdFnGcsPath() != null && options().getJavascriptIdFnName() != null){
+      ValueExtractorFn idFn = ValueExtractorFn.newBuilder()
+      .setFileSystemPath(options().getJavascriptIdFnGcsPath())
+      .setFunctionName(options().getJavascriptIdFnName())
+      .build();
+
+      elasticsearchWriter.withIdFn(idFn);
+    }
```

Comment:
Did you run spotless:apply? The indent seems off

Reply:
yes, it looks like I forgot but I ran it now and checked in the changes